src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
+  const nested = path.join(cwd, 'node_modules', name);
+
+  /* For rnpm installed locally with flat dependencies structure */
+  const flat = path.join(cwd, '..', name);
```

Comment:
In general I agree with you, but here https://github.com/rnpm/rnpm/commit/6cfe40408db8d645aca3b6a0092ecd460d28fd03#diff-d95402e82752076125d62706d2accb1fR18 it wouldn't require a flat one if it's not needed.